src/storage/exec/GetNeighborsNode.h

```diff
@@ -91,8 +91,8 @@ class GetNeighborsNode : public QueryNode<VertexID> {
         int64_t edgeRowCount = 0;
         nebula::List list;
         for (; upstream_->valid(); upstream_->next(), ++edgeRowCount) {
-            if ((limit_ > 0 && edgeRowCount >= limit_) ||
-                (limit_ == 0 && edgeRowCount >= FLAGS_max_edge_returned_per_vertex)) {
+            if ((limit_ >= 0 && edgeRowCount >= limit_) ||
+                (limit_ < 0 && edgeRowCount >= FLAGS_max_edge_returned_per_vertex)) {
```

Comment:
Maybe we could modify the default `limit_` to FLAGS_max_edge_returned_per_vertex, and storage will trust any limit from graph.

Reply:
when graph should not pass a limit -1， return `FLAGS_max_edge_returned_per_vertex `（limit = FLAGS_max_edge_returned_per_vertex） records